src/index.ts

```diff
@@ -73,3 +73,11 @@ export {Text} from './components/text/Text/Text';
 export {TextBase} from './components/text/_private/TextBase/TextBase';
 export {Title} from './components/text/Title/Title';
 export {default as Truncate} from './components/text/Truncate/Truncate';
+export {default as IReactComponentProps} from './common/structures/IReactComponentProps';
+export {
+	IVirtualTableCellRendererDataArgs,
+	IVirtualTableOnChangeRowDataArgs,
+	IVirtualTableRowRendererDataArgs,
+	VirtualTableCellRenderer,
+	VirtualTableRowRenderer,
+} from './components/modules/VirtualTable/VirtualTable';
```

Comment:
Rather than trying to import these types within flywheel-local via `@getflywheel/local-components/src/...`, these are now properly exported for use.